postgres/client.sls

```diff
@@ -25,23 +22,23 @@ postgresql-client-libs:
       - pkgrepo: postgresql-repo
 {%- endif %}
 
+# Alternatives system. Make client binaries available in $PATH
 {%- if 'bin_dir' in postgres %}
+  {% if postgres.linux.altpriority|int > 0 %}
 
-# Make client binaries available in $PATH
-
-  {%- for bin in postgres.client_bins %}
-
-    {%- set path = salt['file.join'](postgres.bin_dir, bin) %}
+    {%- for bin in postgres.client_bins %}
+      {%- set path = salt['file.join'](postgres.bin_dir, bin) %}
 
 {{ bin }}:
   alternatives.install:
     - link: {{ salt['file.join']('/usr/bin', bin) }}
     - path: {{ path }}
-    - priority: 30
+    - priority: {{ postgres.linux.altpriority|int }}
```

Comment:
As well, the `int` filter is unnecessary. YAML renderer does not care about Jinja internal data type.